feat/new product overview pages (#646)
hirosystems/docs: docs/hiro-archive/guides/use-token-metadata-api-archive.md

---

> title: Use Token Metadata API Archive

Reviewer: ```suggestion
title: Use the Token Metadata API Archive
```

> sidebar_label: Token Metadata API Archive

Reviewer: ```suggestion
sidebar_label: Token Metadata API
```

custom_edit_url: null
---

### Prerequisites

Since the Token Metadata API depends on a Stacks Blockchain API, you will need to first launch a Stacks Blockchain API configured for the same Stacks network.

> If you don't already have one, you can follow [these instructions](./use-stacks-blockchain-api-archive.md) to launch one with an archive.

Reviewer: ```suggestion
If you don't already have one, you can follow [these instructions](https://docs/hiro-archive/guides/use-stacks-blockchain-archive) to launch one with an archive.
```

### Where to Download Archives

Token Metadata API archives for each network can be found at the following locations:

- mainnet: https://archive.hiro.so/mainnet/token-metadata-api-pg/
- testnet: https://archive.hiro.so/testnet/token-metadata-api-pg/

The file name patterns are as follows:

- archive: `token-metadata-api-pg-<DATABASE VERSION>-<API VERSION>-<DATE(YYYYMMDD)>.dump`
- shasum: `token-metadata-api-pg-<DATABASE VERSION>-<API VERSION>-<DATE(YYYYMMDD)>.sha256`

There is a continually updated archive and shasum which always points to the most recent upload:

- archive: `token-metadata-api-pg-<DATABASE VERSION>-latest.dump`
- shasum: `token-metadata-api-pg-<DATABASE VERSION>-latest.sha256`

or the most recent upload for a particular version:

- archive: `token-metadata-api-pg-<DATABASE VERSION>-<API VERSION>-latest.dump`
- shasum: `token-metadata-api-pg-<DATABASE VERSION>-<API VERSION>-latest.sha256`

### Restoring the Token Metadata API Using the Hiro Archive

> 1. [Download the archive and shasum for the appropriate network](#where-to-download-archives)

Reviewer: ```suggestion
1. Download the archive and shasum for the appropriate network
```

> 1. [Verify the archive with the shasum](./verify-archive-data.md)

Reviewer: ```suggestion
1. [Verify the archive with the shasum](https://docs/hiro-archive/guides/verify-archive-data)
```
